src/blog/2024/04/node-red-dashboard-2-layout-navigation-styling.md

```diff
@@ -0,0 +1,186 @@
+
+!["Image showing process of setting page layout"](./images/node-red-dashboard-2-layout-navigation-styling-setting-new-page-layout.gif "Image showing process of setting page layout"){data-zoomable}
+
+## Setting Dashboard 2.0 elements size
+
+Setting the size for elements in Dashboard 2.0 is straightforward, but understanding the actual unit size in the size property can be a bit tricky. It's important to note that the size of a horizontal unit varies depending on the layout, while the vertical size is fixed at **48px** currently.
+
+## Understanding unit size in different layouts
+
+#### Grid
+
+In a grid layout, each unit specified in the size property represents a column, dividing the page evenly. When you assign columns to a group, it forms an internal grid with the same number of columns as the group. For example, if you have a group with 4 columns and two widgets inside it, each widget can be sized by considering those columns. If you set 1 column for the first widget and 3 columns for the second widget, the first widget will occupy 25% of the group's width, and the second widget will occupy 75%.
+
+#### Notebook
+
+In the Notebook layout, As you know the width for the group is fixed at **1024px**. The size unit represents the number of columns that the group will create or represent in its internal grid, increasing size will increase the number of columns in group. For example, if you have one group with a size of 12 columns, that group will create an internal grid with 12 columns, and widgets can occupy those columns using the size property where each will also represent one column in group.
+
+#### Fixed
+
+The fixed layout in Dashboard 2.0 employs a flexbox design with a single row of widgets. Each width unit in this layout is set at a fixed 90px. Groups follow the same internal grid pattern as other layouts. When a widget occupies the entire screen width, it shifts to the next row. However, this layout remains unchanged for varying screen sizes, potentially resulting in empty space on larger screens.
+
+###  Setting element size 
+
+To set the size of groups and widgets in Dashboard 2.0, follow these steps:
+
+1. Go to the Dashboard 2.0 sidebar and click on the edit button next to the element you want to resize.
+2. Adjust the size using the size property.
+
+!["Image showing process of setting element size"](./images/node-red-dashboard-2-layout-navigation-styling-setting-size.gif "Image showing process of setting element size"){data-zoomable}
+
+## Understanding Dashboard 2.0 Theme 
+
+The theme is a collection of colors that control the look and feel of the widgets, groups, and other elements on the page.
+
+In Dashboard 2.0, while adding a page ( ui-page ) we have to specify which theme it will use. By default, we have one default theme in dashboard 2.0. Additionally, we can add a new theme (ui-theme) where we need to specify colors and sizing.
```

Comment:
```suggestion
In Dashboard 2.0, when adding a page ( ui-page ) we can specify which theme it will use. By default, we have one theme in dashboard 2.0, we can add more themes using the Dashboard 2.0 side panel.
```